src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/processedenvelopes/ProcessedCcdType.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.processedenvelopes;
+
+public enum ProcessedCcdType {
```

Comment:
maybe `EnvelopeCcdAction` or `EnvelopeCcdResult`?

Reply:
in processor we used  ProcessedCcdType, but  EnvelopeCcdAction sound good to me

Reply:
changed to EnvelopeCcdAction